src/daemon/global_rpc/server.rs

```diff
@@ -98,4 +99,19 @@ impl GlobalRpcProtocol for GlobalRpcImpl {
         }
         None
     }
+
+    async fn alloc_forward(&self, registration: ForwardRequest) -> bool {
```

Comment:
It might be better to return a Result rather than a bool. It's always a bit confusing whether true is fail or false is fail